pkg/cmd/server/api/validation/master.go

```diff
@@ -287,6 +287,14 @@ func ValidateAssetConfig(config *api.AssetConfig) fielderrors.ValidationErrorLis
 		allErrs = append(allErrs, urlErrs...)
 	}
 
+	if _, loggingURLErrs := ValidateURL(config.LoggingPublicURL, "loggingPublicURL"); len(loggingURLErrs) > 0 {
```

Comment:
`ValidateURL` returns errors fails on zero length strings.  Also, should you have warnings when these aren't present even though its not strictly required?